boot-qemu-help.txt

```diff
@@ -0,0 +1,42 @@
+Usage: ./boot-qemu.sh <options>
+
+Script description: Boots a Linux kernel in QEMU.
+
+Required parameters:
+  -a | --arch | --architecture:
+    The architecture to boot. Possible values are:
+       * arm32_v5
+       * arm32_v6
+       * arm32_v7
+       * arm64
+       * mips
+       * mipsel
+       * ppc32
+       * ppc64
+       * ppc64le
+       * x86_64
+
+  -k | --kbuild-folder:
+    The kernel build folder, as an absolute path or relative path
+    from wherever the script is being run. This is wherever the
+    compiled vmlinux image lives, not the architecture's boot folder.
```

Comment:
Why the folder and not path to kernel image?

Reply:
For a couple of the ARM devices, we need the dtb to boot:

https://github.com/nathanchance/boot-utils/blob/101ef4f4258719dd4879cf68ae28af8cc9351595/boot-qemu.sh#L97

https://github.com/nathanchance/boot-utils/blob/101ef4f4258719dd4879cf68ae28af8cc9351595/boot-qemu.sh#L105